app/uk/gov/hmrc/tai/model/nps/NpsTaxAccount.scala

```diff
@@ -101,15 +100,20 @@ case class NpsTaxAccount(nino: Option[String],
 
   private[nps] def getEmploymentCeasedDetail(npsEmployments : List[NpsEmployment]) = {
 
+
+    val minusOneDay = TaxYear().start.minusDays(1)
```

Comment:
This is TaxYear().prev.end
I think the variable name may need updating either way

Reply:
Fixed